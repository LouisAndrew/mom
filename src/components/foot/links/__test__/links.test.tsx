import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';

import Links from '..';
describe('Links on footer component', () => {
    const el: React.ReactElement = <Links />;

    afterEach(cleanup);

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(el, div);
    });
    it('matches snapshot', () => {
        const tree = renderer.create(el).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
