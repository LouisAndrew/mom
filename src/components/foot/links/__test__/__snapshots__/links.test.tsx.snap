// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links on footer component matches snapshot 1`] = `
<ul
  className="sc-AxheI btopJg"
>
  <li
    className="sc-Axmtr epJlBq"
    fontFamily="heading"
    fontSize={
      Array [
        2,
        2,
      ]
    }
  >
    <a
      aria-current="page"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
  </li>
  <li
    className="sc-Axmtr epJlBq"
    fontFamily="heading"
    fontSize={
      Array [
        2,
        2,
      ]
    }
  >
    <a
      aria-current="page"
      href="/"
      onClick={[Function]}
    >
      Katalog
    </a>
  </li>
</ul>
`;
